docs/SPEC.md

```diff
@@ -1250,6 +1250,9 @@ Example:
 
     from(bucket:"telegraf/autogen")
     from(bucketID:"0261d8287f4d6000")
+    
+__NOTE__  
+The default group key for the tables produced by `from` is every column except `_time` and `_value`.
```

Comment:
Can you move this line up with the rest of the description of the `from` function? That way it does not need to be its own NOTE but rather is just part of the normal description of the `from` function.